.github/workflows/rust.yml

```diff
@@ -370,34 +372,48 @@ jobs:
     runs-on: ${{ matrix.os }}
     container: ${{ matrix.container }}
     steps:
-      - name: Checkout repository
+      - name: Checkout ockam cli repository
         uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
         with:
-          ref: ${{ github.event.inputs.release_branch }}
+          ref: ${{ inputs.ockam_command_cli_version != '' && inputs.ockam_command_cli_version || inputs.commit_sha  }}
+          path: ockam_cli
+
+      - name: Checkout ockam bats repository
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+        with:
+          ref: ${{ inputs.commit_sha }}
+          path: ockam_bats
 
       - name: Use cache cargo_home
-        uses: ./.github/actions/cargo_home_cache
+        uses: ./ockam_bats/.github/actions/cargo_home_cache
       - name: Use cache cargo_target
-        uses: ./.github/actions/cargo_target_dir_cache
+        uses: ./ockam_bats/.github/actions/cargo_target_dir_cache
+        with:
+          ockam_dir: ockam_cli
 
       - name: Build Binary
         shell: bash
+        working-directory: ockam_cli
         run: |
           rustc --version
           set -x
           cargo build --bin ockam
+          echo "PATH=$(pwd)/target/debug:$PATH" >> $GITHUB_ENV;
 
       - name: Run Script On Ubuntu
-        if: matrix.build == 'linux_86'
+        working-directory: ockam_bats
```

Comment:
It's deliberate that we don't limit this one on the matrix key anymore, right?

Reply:
Yeah, we only run against a single matrix build